packages/shared/src/components/notifications/InAppNotification.module.css

```diff
@@ -0,0 +1,32 @@
+.inAppNotificationAvatar {
+    img {
+        border-radius: 6px;
+        height: 2rem;
+        width: 2rem;
+    }
+
+    svg {
+        height: 1.5rem;
+        width: 1.5rem;
+    }
+}
+
+@keyframes inAppNotificationAnimationEntry {
+  from {
+    transform: translateX(10rem);
```

Comment:
Perhaps it's better to use percentage here?

Reply:
Fair point let me update that.

Reply:
done